src/library-authoring/routes.ts

```diff
@@ -33,10 +33,10 @@ export const ROUTES = {
   COLLECTION: '/collection/:collectionId/:selectedItemId?',
   // LibrarySectionPage route:
   // * with a selected sectionId and/or an optionally selected subsectionId.
-  SECTION: '/section/:sectionId/:subsectionId?',
+  SECTION: '/section/:sectionId/:selectedItemId?',
   // LibrarySubsectionPage route:
   // * with a selected subsectionId and/or an optionally selected unitId.
-  SUBSECTION: '/subsection/:subsectionId/:unitId?',
+  SUBSECTION: '/subsection/:subsectionId/:selectedItemId?',
```

Comment:
To do: can we avoid parsing out `sectionId` and `subsectionId` and just have a `containerId`?

Reply:
Addressed with 5cecfa8ab9fb034be314fbb17db5ddff619f9661.